app/src/main/java/com/pysiak/simple/clock/ui/SimpleClockView.java

```diff
@@ -0,0 +1,16 @@
+package com.pysiak.simple.clock.ui;
+
+import android.support.v4.app.Fragment;
+
+import java.util.List;
+
+/**
+ * This is interface for communicating SimpleClockActivity with ClockPresenter
```

Comment:
Wrong desciption. Realization of you interface could be different (not SimpleClockActivity)

Need something like:
Defines contract/interface for main clock screen.